### DISCUSSION TITLE
No access to students pack after academic status is already verified for a week

### DISCUSSION THREAD

**Discussion by prithvi-raj-bagoria** [community/community#111799] [GitHub Education]
### Hi everyone,

My academic status has already been verified a week ago but i am not getting academic benefits. Whenever i go to verification site
it shows  : "You were last verified as a null on Mar 7, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits."

**Comment by ngljcb** [comment]
If you've already been verified for academic status on GitHub but haven't received the benefits yet, wait for a while as there might be a waiting period. Check periodically for updates in your account settings. If the issue persists, contact GitHub support for assistance.

**Comment by saifeddineelhanoune** [comment]
the features of student pack claiming should have a valid school email and to configure it with github before claim you pack
